src/main/java/org/pesmypetcare/webservice/dao/WeightDaoImpl.java

```diff
@@ -0,0 +1,168 @@
+package org.pesmypetcare.webservice.dao;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.QuerySnapshot;
+import org.pesmypetcare.webservice.entity.WeightEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.pesmypetcare.webservice.firebaseservice.FirebaseFactory;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+@Repository
+public class WeightDaoImpl implements WeightDao {
+    private final String DELFAIL_KEY;
```

Comment:
Constants should be static final and be initialized inline

Reply:
Solved!